lib/runtime/wasmer/exports_test.go

```diff
@@ -1322,7 +1324,23 @@ func TestInstance_ParachainHostValidationCode(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEmpty(t, validationCode)
 
-	expected := []byte(parachainTestData.Expected["validationCode"])
+	expected := []byte(parachainTestData.Lookups["validationCode"])
+	require.Equal(t, expected[4:], []byte(*validationCode))
+}
+
+func TestInstance_ParachainHostValidationCodeByHash(t *testing.T) {
```

Comment:
Could you move this test from `wasmer` to `wazer` since we are about to remove this package from the project?